### ISSUE TITLE
match locations_computed filtering to number_of_turbines filtering

### ISSUE THREAD

**Issue by rjhanes** [NREL/celavi#1]
After the addition of filtering on turbines and integrating it within the compute locations, the only facility that still shows up extra in number of turbines is 55871. While this is an easy fix, would you like to delve deeper into why this survived?

@TJTapajyoti Commit 2767c0a may resolve this issue, can you confirm?

**Comment by rjhanes** [comment]
@TJTapajyoti I need to revise some things in the `DataManager` so I'll also take a look at revising the `TurbineLocations` class so it'll work for generating both locations and number_of_turbines.

**Comment by rjhanes** [comment]
I found that at commit 842cfbe in `develop`, computing the locations and the number_of_turbines file resulted in 1,141 unique power plant `facility_id` values in locations_computed but only 1,063 unique facilities in number_of_turbines. I'm going to apply the more stringent filtering used to generate number_of_turbines to both datasets, to avoid having unused facilities in `CostGraph` and extra routes being computed.

**Comment by rjhanes** [comment]
I tested commit 06ba030 on the `national_data` branch and confirmed that both locations_computed and number_of_turbines wound up with an identical set of `facility_id` values for the power plant facility type, and no duplicate entries in locations_computed. I made these changes in the `issue23` branch (it's related to the capacity expansion additions) so I'll merge that branch into `develop` but keep the `issue23` branch active for further development. I also pushed the updated locations_computed and number_of_turbines files to the `national_data` branch.
